spec/helpers/session.rb

```diff
@@ -0,0 +1,20 @@
+module SessionHelpers
+
+  def sign_up(user)
+    visit '/users/new'
+    fill_in :email, with: user.email
+    fill_in :password, with: user.password
+    fill_in :first_name, with: user.first_name
+    fill_in :last_name, with: user.last_name
+    fill_in :username, with: user.username
+    click_button 'Sign up'
+  end
+
+  def sign_in(email:, password:)
```

Comment:
Unused method argument - `email`. You can also write as `sign_in(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `password`. You can also write as `sign_in(*)` if you want the method to accept any arguments but don't care about them.